### PR TITLE
Dra 436 enable oauth2

### PR DESCRIPTION
**Pull request by thomasegense**
All OAuth classes and config added.
Enabled OAuth only on method /record/{id}

### CODE REVIEW

File: src/main/openapi/ds-present-openapi_v1.yaml [hunk 1]
```diff
@@ -1115,18 +1118,56 @@ components:
...
+ # TODO: Inject this from properties when DRA-139 has been implemented
+  securitySchemes:
+
+    KBOAuth:
+      type: oauth2
+      description: 'This API uses OAuth 2 with the implicit grant flow. [More info](https://api.example.com/docs/auth)'
+      flows:
+        implicit:   # <---- OAuth flow(authorizationCode, implicit, password or clientCredentials)
+          authorizationUrl: 'https://keycloak-devel-01.kb.dk/realms/ds-poc/protocol/openid-connect/auth'
+          #authorizationUrl: https://api.example.com/oauth2/authorize # Overridden by the configuration
```
Comment (VictorHarbo) [inline src/main/openapi/ds-present-openapi_v1.yaml:1158]: Please inject from properties with syntax ${config.yaml.path}

**Review by VictorHarbo — CHANGES_REQUESTED** [review]
Looks good. Only one comment here.

File: src/main/openapi/ds-present-openapi_v1.yaml (hunk 1 above)
Comment (thomasegense) [reply src/main/openapi/ds-present-openapi_v1.yaml]: fixed

**Review by VictorHarbo — APPROVED** [review]
Looks good now

**Pull request merged**